### PR TITLE
Wss/test

### PR DESCRIPTION
**Pull request by wss-git**
- 新增测试能力
- publish yaml 参数完善
- 新增 eslist 能力
- 移除国际化（ i18n）能力

### CODE REVIEW

File: .signore [hunk 1]
```diff
@@ -4,4 +4,6 @@ tsconfig.json
 LICENSE
 package-lok.json
 .prettierrc.js
-.gitignore
\ No newline at end of file
+.gitignore
+coverage
```
Comment (SquatsTonight) [inline .signore:8]: test 目录是否需要加在这里

File: package.json [hunk 2]
```diff
@@ -36,16 +40,24 @@
     "i18n": "^0.13.2",
```
Comment (SquatsTonight) [inline package.json]: i18n 这种不必要的依赖可以去掉

File: test/other.test.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import loggerStarter from '../src/common/logger';
```
Comment (SquatsTonight) [inline test/other.test.ts]: 这个文件名称太随意了，从文件名看不出来要测啥

File: .signore (hunk 1 above)
Comment (wss-git) [reply .signore:8]: 这个和ncc 一起改吧，ncc 那个 signore 比较全

File: test/index-integration.test.ts [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+  })
+
+  it('publish cli', async () => {
+    const inp = _.cloneDeep(inputs);
+    inp.props = {};
+    inp.args = `--region cn-shenzhen --layer-name ${random} --code ${path.join(__dirname, 'test.zip')}`;
+
+    const componentStarter = new ComponentStarter();
+    const result = await componentStarter.publish(inp);
+    const reg = new RegExp(`#${random}#1$`);
```
Comment (SquatsTonight) [inline test/index-integration.test.ts:53]: 这些都是 happy case 啊，增加一些错误的 case，例如删除一个不存在的 layer 应该出现什么表现，包括 ut 也需要加一些这种 corner case

File: test/index-integration.test.ts [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+    props: {
+      region: 'cn-shenzhen',
+      layerName: random,
+    },
+    credentials: {
+      AccountID: process.env.AccountID,
+      AccessKeyID: process.env.AccessKeyID,
+      AccessKeySecret: process.env.AccessKeySecret,
+    },
+    appName: 'fc-sync-test',
```
Comment (alapha23) [inline test/index-integration.test.ts]: `fc-layer-test`?
